spec/consensus/proposer-based-timestamp/pbts-sysmodel.md

```diff
@@ -157,40 +124,38 @@ For PBTS, a `proposal` is a tuple `(v, v.time, v.round)`, where:
 - `v.round` is the round at which `v` was first proposed.
 
 We include the proposal round `v.round` in the proposal definition because a
-value `v` and its associated proposal time `v.time` can be proposed in multiple
-rounds, but the evaluation of the `timely` predicate is only relevant at round
-`v.round`.
+value `v` can be proposed in multiple rounds of consensus,
+but the evaluation of the `timely` predicate is only relevant at round `v.round`.
 
 > Considering the algorithm in the [arXiv paper][arXiv], a new proposal is
-> produced by the `getValue()` method, invoked by the proposer `p` of round
-> `round_p` when starting its proposing round with a nil `validValue_p`.
-> The first round at which a value `v` is proposed is then the round at which
-> the proposal for `v` was produced, and broadcast in a `PROPOSAL` message with
-> `vr = -1`.
+> produced by the `getValue()` method (line 18), invoked by the proposer `p` of round
+> `round_p` when starting the round with `validValue_p = nil`.
+> In this case, the proposed value is broadcast in a `PROPOSAL` message with
+> `vr = validRound_p = -1`.
 
 #### **[PBTS-PROPOSAL-RECEPTION.0]**
 
 The `timely` predicate is evaluated when a process receives a proposal.
 More precisely, let `p` be a correct process:
 
 - `proposalReceptionTime(p,r)` is the time `p` reads from its local clock when
-  `p` is at round `r` and receives the proposal of round `r`.
+  it receives the proposal of round `r`.
```

Comment:
Notice I've removed the requirement of `p` being at round `r`. The goal is to allow the early reception of proposals, which does not impact the algorithm.